content/docs/deployment/csmoperator/release/_index.md

```diff
@@ -32,12 +33,15 @@ Description: >
 - [#1143 - [BUG]: CSM Operator does not calculate status correctly when deploying the authorization proxy server](https://github.com/dell/csm/issues/1143))
 - [#1146 - [BUG]: CSM Operator does not calculate status correctly when deploying observability with csi-powerscale](https://github.com/dell/csm/issues/1146))
 - [#1147 - [BUG]: CSM Operator labels csm objects with CSMVersion 1.8.0, an old version](https://github.com/dell/csm/issues/1147))
+- [#1156 - [BUG]: CSM object in success state when all CSI Powerflex pods are failing due to bad secret credentials](https://github.com/dell/csm/issues/1156))
+- [#1157 - [BUG]: If Authorization Proxy Server is installed in an alternate namespace by CSM Operator, the deployment fails](https://github.com/dell/csm/issues/1157))
+- [#1159 - [BUG]: CSM status is not always accurate when Observability installed with only one or two components](https://github.com/dell/csm/issues/1159))
 
 ### Known Issues
 | Issue | Workaround |
 |-------|------------|
-| The status calculation done for the csm object associated with the Authorization Proxy Server when deployed with CSM Operator assumes that the proxy server will be deployed in the "authorization" namespace. If a different namespace is used, the status will stay in the failed state, even though the deployment is healthy. | We recommend using the "authorization" namespace for the proxy server. If this is not possible, the health of the deployment can be verified by checking the status of all the pods rather than by checking the status field.|
-| The status field of a csm object as deployed by CSM Operator may, in limited cases, display a "Failed" status for a successful deployment. | As a workaround, the deployment is still usable as long as all pods are running/healthy. |
+| The status field of a csm object as deployed by CSM Operator may, in limited cases, display an incorrect status for a deployment. | As a workaround, the health of the deployment can be determined by checking the health of the pods. |
+| When CSM Operator creates a deployment that includes secrets (e.g., application-mobility, observability, cert-manager, velero), these secrets are not deleted on uninstall and will be left behind. For example, the `karavi-topology-tls`, `otel-collector-tls`, and `cert-manager-webhook-ca` secrets will not be deleted. | This should not cause any issues on the system, but all secrets present on the cluster can be found with `kubectl get secrets -A`, and any unwanted secrets can be deleted with `kubectl delete secret -n <secret-namespace> <secret-name>`|
```

Comment:
Any links to tracking GitHub tickets?

Reply:
@AronAtDell I don't think we have GitHub tickets for either yet, just Jira tickets.

Reply:
O.K. Once you have the JIRA tickets that is good. The users may also want to see corresponding GitHub issues as you will need those in any event when you address the known issue.